lib/server.js

```diff
@@ -102,9 +102,9 @@ function parsePOST(req) {
 
 function logResponse(opts, response, startTime) {
     const latency = Date.now() - startTime;
-    let logLevel = 'trace/request';
+    let logLevel = 'info/request';
```

Comment:
This will log the response together with the stack. What we want is the incoming request together with the response code and latency.